### DISCUSSION TITLE
Security check with Admin "Ok" (green) - with User "Error" (red)!

### DISCUSSION THREAD

**Discussion by Rimos911** [nextcloud/all-in-one#5272] [Questions] [answered]
I noticed that when I run the security check with my Nextcloud admin account I get a green tick (everything is OK!) but under my user, to whom I have given permission to also check, I get a red error message (same instanz)!

situation under my User:
![Bildschirmfoto vom 2024-09-16 16-35-55](https://github.com/user-attachments/assets/745a2d9b-e1cc-45a2-a5bb-2adba86ff344)

Situation under Admin:
![Bildschirmfoto vom 2024-09-16 16-39-41](https://github.com/user-attachments/assets/3359c255-0766-4529-89df-be9a82ca2b7f)

I use Nextcloud 30.0 AIO (Docker)

**Comment by szaimen** [comment, marked as answer]
Hi, this looks like a bug in nextcloud/server. Please open a bug report here: https://github.com/nextcloud/server
